tests/x11/toolkits/fltk.pm

```diff
@@ -21,8 +41,8 @@ sub run {
     script_run './fltk', 0;
     assert_screen [qw(ui-toolkit-fltk ui-toolkit-fltk-nomsg-display)];
     if (match_has_tag 'ui-toolkit-fltk-nomsg-display') {
+        move_window;
```

Comment:
we need to know if packagehub is available

Reply:
Updated the code: add_suseconnect_product(get_addon_fullname('phub')) if (is_phub_ready());